Toolchain/BuildPython.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e
+
+# This file will need to be run in bash, for now.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+echo "$DIR"
+
+TARGET=i686-pc-serenity
+PREFIX="$DIR/Local"
+SYSROOT="$DIR/../Root"
+
+source ../Ports/python-3.6/version.sh
+
+echo PYTHON_VERSION is "$PYTHON_VERSION"
+echo PYTHON_URL is "$PYTHON_URL"
+
+echo PREFIX is "$PREFIX"
+echo SYSROOT is "$SYSROOT"
+
+mkdir -p "$DIR/Tarballs"
+
+source "$DIR/UseIt.sh"
+
+pushd "$DIR/Tarballs"
+    if [ ! -e "$PYTHON_ARCHIVE" ]; then
+        curl -O "$PYTHON_URL"
+    else
+        echo "Skipped downloading Python-$PYTHON_VERSION"
+    fi
+
+    md5="$(md5sum $PYTHON_ARCHIVE | cut -f1 -d' ')"
+    echo "python md5='$md5'"
+    if  [ "$md5" != "$PYTHON_MD5SUM" ] ; then
+        echo "python md5 sum mismatching, please run script again."
+        rm $PYTHON_ARCHIVE
+        exit 1
+    fi
+
+    if [ ! -d "Python-$PYTHON_VERSION" ]; then
+        echo "Extracting python..."
+        tar -xf "$PYTHON_ARCHIVE "
```

Comment:
Extra whitespace here at the end of the arg, so `tar` ends up being unable to open the file.

Reply:
True! Was a mistake from refactoring. =)